extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/KubernetesClientProcessor.java

```diff
@@ -68,16 +63,15 @@ public class KubernetesClientProcessor {
     private static final DotName WATCHER = DotName.createSimple(Watcher.class.getName());
     private static final DotName RESOURCE_EVENT_HANDLER = DotName
             .createSimple(io.fabric8.kubernetes.client.informers.ResourceEventHandler.class.getName());
-    private static final DotName KUBERNETES_RESOURCE = DotName.createSimple(KubernetesResource.class.getName());
-    private static final DotName VALIDATION_SCHEMA = DotName.createSimple(ValidationSchema.class.getName());
+    private static final DotName KUBERNETES_RESOURCE = DotName
+            .createSimple(KubernetesResource.class.getName());
     private static final DotName KUBERNETES_RESOURCE_LIST = DotName
             .createSimple(KubernetesResourceList.class.getName());
-    private static final DotName KUBE_SCHEMA = DotName.createSimple(KubeSchema.class.getName());
     private static final DotName VISITABLE_BUILDER = DotName.createSimple(VisitableBuilder.class.getName());
+    private static final DotName VALIDATION_SCHEMA = DotName.createSimple(ValidationSchema.class.getName());
+    private static final DotName KUBE_SCHEMA = DotName.createSimple(KubeSchema.class.getName());
```

Comment:
Nitpick: Are these changes applied automatically by formatting? Can you please avoid them?

Reply:
Done!

Reply:
Thanks!